internal/database/service_test.go

```diff
@@ -6,7 +6,7 @@ import (
 	"testing"
 
 	"github.com/neo4j/mcp/internal/database"
-	"github.com/neo4j/mcp/internal/database/mocks"
+	database_mocks "github.com/neo4j/mcp/internal/database/mocks"
```

Comment:
nit: should we stick to the convention for package names so it reads a bit better https://go.dev/doc/effective_go#package-names

`database_mocks.NewMockService(ctrl)` reads a bit weird imo
my suggestion is we can important them as `db` or even `database` so it reads like this in tests:
```
database.NewMockService(ctrl) or db.NewMockService(ctrl)
analytics.NewMockService(ctrl)
```
just anything but snake case please:)
note: file names are ok, for some reason